activerecord/test/cases/store_test.rb

```diff
@@ -148,6 +148,20 @@ class StoreTest < ActiveRecord::TestCase
     assert_equal "Dallas", @john.partner_name_before_last_save
   end
 
+  test "saved changes tracking for accessors with json column" do
+    if current_adapter?(:Mysql2Adapter) && ActiveRecord::Base.connection.mariadb?
```

Comment:
```suggestion
    if current_adapter?(:Mysql2Adapter, :TrilogyAdapter) && ActiveRecord::Base.connection.mariadb?
```

Reply:
This addition will make the tests pass with the introduction of the `TrilogyAdapter`

Reply:
@rdimartino Might be worth making the change if this is still something we want to get merged

Reply:
Thanks!